TradeItIosTicketSDK2/TradeItPortfolioEquityPositionPresenter.swift

```diff
@@ -53,11 +50,11 @@ class TradeItPortfolioEquityPositionPresenter: TradeItPortfolioPositionPresenter
         return NumberFormatter.formatPercentage(totalGainLossPercentage.floatValue);
     }
 
-    func getCostBasis() -> String {
-        guard let costBasis = self.position?.costbasis
+    func getAvgCostBasis() -> String {
```

Comment:
Is there a different meaning in `getAvgCostBasis` vs `getAvgCost`? If there is no difference, then we should stick with something consistent like `getAvgCost`.